[WPT] Add Test-import commit class

JonWBedard opened the pull request:
#### f044b634bcf46d95489340c618fe1f05c76c735c
<pre>
[WPT] Add Test-import commit class
<a href="https://bugs.webkit.org/show_bug.cgi?id=258575">https://bugs.webkit.org/show_bug.cgi?id=258575</a>
rdar://111392835

Reviewed by Aakash Jain.

* metadata/commit_classes.json: Add Test-import class.

Canonical link: <a href="https://commits.webkit.org/266107@main">https://commits.webkit.org/266107@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/950598ca9896aed1a46fcf012bcb0c23e452bfe2

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/12889 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/13215 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/13544 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/14629 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/12306 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/15717 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/13236 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/15002 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/13055 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/13770 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/10909 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/15080 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/11058 "Passed tests") | | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/18727 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/12133 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/11829 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/15029 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/12293 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/10192 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/11554 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/15870 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/1464 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/12133 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

gsnedders commented on metadata/commit_classes.json:
```diff
@@ -53,6 +53,17 @@
+		"paths": [
+			"LayoutTests/TestExpectations",
+			"LayoutTests/tests-options.json",
+			"LayoutTests/platform",
```
Can we use wildcards? Because it would be kinda nice to have this be `LayoutTests/platform/*/imported`.

JonWBedard replied:
We can, as it turns out! Let me modify this....

gsnedders reviewed (commented):
non-reviewer r+, but am curious as to whether we can do better with the paths here

JonWBedard committed:
[WPT] Add Test-import commit class
https://bugs.webkit.org/show_bug.cgi?id=258575
rdar://111392835

Reviewed by Aakash Jain.

* metadata/commit_classes.json: Add Test-import class.

Canonical link: https://commits.webkit.org/266107@main
